### ISSUE TITLE
Check Globus tokens and endpoints before creating tarballs

### ISSUE THREAD

**Issue by lukaszlacinski** [E3SM-Project/zstash#172]
Currently, zstash creates a tarball first, and then tries to transfer it to a Globus endpoint, if --hpss option is set to a Globus URL. It usually takes several minutes to create a default-size tarball, and then zstash exits if Globus endpoints are not activated. Zstash should check if --hpss points to a Globus URL and tries to get a Globus token and verify if Globus endpoints are activated before it starts creating tarballs.

**Comment by golaz** [comment]
This would be nice to have at some point, but we can live with what we have for now. 

For new users, we should tell them to first do a small test transfer for the initial configuration to take place. For subsequent transfers, they should make sure to activate nodes first before starting zstash.

@forsyth2 : are you planning to update the documentation?

**Comment by forsyth2** [comment]
> @forsyth2 : are you planning to update the documentation?

@golaz Do you mean for this issue? @lukaszlacinski included documentation updates in #154. If you want to include your suggestion for new users now, it would probably be best to include that in #154 too.

**Comment by golaz** [comment]
I added a comment in https://github.com/E3SM-Project/zstash/pull/154/files#diff-66e28562c85a1bb96f431f2dd52bbabab914359e4b06f58eb1e18eadc01f5c06

**Comment by forsyth2** [comment]
Resolved by #185/#214.
